src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -222,7 +222,7 @@ export class ProjectTokenHistoricalPriceService {
      * calculated forecast to the end of the array up to the current time.
      */
     const forecastData = await this.getTrajectoryForecastData(lbpMgr);
-    const pastForecast = forecastData?.filter((item) => item.time < (new Date().getTime() / 1000));
+    const pastForecast = forecastData?.filter((item) => item.time < (currentTime));
```

Comment:
as long as we're making changes, `(currentTime)` would look a little less silly without the parens.